src/platform-includes/session-replay/install/javascript.angular.mdx

```diff
@@ -0,0 +1,37 @@
+```bash {tabTitle:npm}
+# Angular 12 and newer:
+npm install --save @sentry/angular-ivy
```

Comment:
is this a new sdk name? we might need to include it in some lists in the app too

Reply:
Yeah, it's nuget trends uses too: https://github.com/dotnet/nuget-trends/blob/3b12f722505aca5423e469449549e46ab4986f6c/src/NuGetTrends.Web/Portal/package.json#L25-L26

Where do we need to include it?

Reply:
https://github.com/getsentry/sentry/blob/eee482eb315cc0bdae53dd0bddc96c95e7dc2f27/static/app/data/platformCategories.tsx#L330-L372 for sure.

I looked at some related onboarding/docs stuff... we might need to also create `angular-ivy-replay-onboarding-1-install.md` in the docs site too. I think it's meant to be moved into sentry repo, but i can't find the code here right now.

Reply:
Oh that, It's already pointing to the right package: 
![image](https://github.com/getsentry/sentry-docs/assets/1633368/e45357ea-98ca-4fd9-a37b-3b6ea1ef113e)